src/index.scss

```
@@ -44,6 +44,7 @@
 @import "scss/historyStats.scss";
 @import "scss/liveDraft.scss";
 @import "scss/segmentedToggle.scss";
+@import "scss/profileView.scss";
```

Stylelint (17.14.0)

[error] Unexpected extension ".scss" in `@import` (scss/load-partial-extension)

(scss/load-partial-extension)